gpflow/monitor.py

```diff
@@ -304,7 +304,7 @@ class Monitor:
         log_dir = "logs"
         model_task = ModelToTensorBoard(log_dir, model)
         image_task = ImageToTensorBoard(log_dir, plot_prediction, "image_samples")
-        lml_task = ScalarToTensorBoard(log_dir, lambda: model.log_likelihood(), "lml")
+        lml_task = ScalarToTensorBoard(log_dir, lambda: model.log_marginal_likelihood(), "lml")
```

Comment:
there are `log_marginal_likelihood`s for some but not all models. is that OK for this example?

Reply:
I think it's fine, it's just an example